go/api/response_handlers.go

```diff
@@ -123,17 +123,17 @@ func parseMap(response *C.struct_CommandResponse) (interface{}, error) {
 		return nil, nil
 	}
 
-	value_map := make(map[interface{}]interface{}, response.array_value_len)
+	value_map := make(map[string]interface{}, response.array_value_len)
```

Comment:
Just for my understanding, are we certain that all the `map` use cases only have `string` as it's key?

Reply:
Yes. If that changes, we get big troubles with other clients.